templates/rustdoc/topbar.html

```diff
@@ -226,7 +226,25 @@
                 </li>
             {%- endfor -%}
         </ul>
-    </li>
+    </li>{#
+    Display the features available in current build
+    #}
+    {% if 'krate' in __tera_context %}
+        <li class="pure-menu-item pure-menu-has-children">
+            <a href="#" class="pure-menu-link">
+                <span class="title">Feature flags</span>
+            </a>
+
+            {# Build the dropdown list showing available features #}
+            <ul class="pure-menu-children">
+                {%- for feature in krate.features -%}
+                    <li class="pure-menu-item">
+                        <a href="#" class="pure-menu-link">{{ feature }}</a>
```

Comment:
Is there a reason to make these links? It seems weird that clicking on them does nothing.

Reply:
Frankly I was not sure about style of other things, but yes it looks weird, done.